html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md

```diff
@@ -0,0 +1,29 @@
+### Introduction
+This project is intended to give you a chance to flex some of the new items you've been absorbing over the past few lessons. This time it's a sign-up form for an imaginary service.
+
+![project](./sign-up-form.png)
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+#### Step 1: Set up and planning
+
+1. Set up your HTML and CSS files with some simple dummy content, just to make sure you have everything linked correctly.
+2. Set up your git repository (refer to past projects if you need a refresher).
+3. Download a full-resolution copy of [the design file](INSERT LINK HERE), and get a general idea for how you're going to need to lay things out in your HTML document.
+
+#### Step 2: Gather Assets
+
+1. The design has a large background-image, so go find and download an image you want to use for that section. The one in the design can be found [on unslpash.com](https://unsplash.com/photos/25xggax4bSA), but feel free to select your own. Be sure to credit the creator of your image!
```

Comment:
Need Oxford coma after "find" (personal preference)

Reply:
🤔 isn't the oxford comma for lists?

Reply:
i'll excuse myself, no need to escort me